src/grt/src/MakeWireParasitics.cpp

```diff
@@ -331,8 +341,17 @@ MakeWireParasitics::routeLayerLengths(odb::dbNet* db_net)
     Net* net = grouter_->getNet(db_net);
     for (Pin& pin : net->getPins()) {
       int layer = pin.getConnectionLayer() + 1;
-      const odb::Point& grid_pt = pin.getOnGridPosition();
-      const odb::Point& pt = pin.getPosition();
+      odb::Point grid_pt = pin.getOnGridPosition();
+      odb::Point pt = pin.getPosition();
+
+      std::vector<std::pair<odb::Point, odb::Point>> ap_positions;
+      bool has_access_points = grouter_->pinAccessPointPositions(pin, ap_positions);
+      if (has_access_points) {
+        auto ap_position = ap_positions.front();
```

Comment:
It would be better to use the preferred access point rather than just the first one.

Reply:
This is ok I just missed it.